### PR TITLE
Add OmenDisputeResolution Generic Scheme

### PR DESCRIPTION
**Pull request by nicoelzer**
This commit adds a GenericScheme to integrate the DxDAO with realit.io as arbitrator to resolve disputes. More information can be found:

- https://daotalk.org/t/omen-mvp-overview/1229 (Section Arbitration)
- https://realitio.github.io/docs/html/arbitrators.html

This PR provides the GenericScheme for Kovan & Rinkeby for testing purposes. Mainnet will be added later.

### CODE REVIEW

File: src/genericSchemeRegistry/index.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ const ensRegistryInfo = require("./schemes/ENSRegistry.json");
 const ensPublicResolverInfo = require("./schemes/ENSPublicResolver.json");
 const registryLookupInfo = require("./schemes/RegistryLookup.json");
 const co2kenInfo = require("./schemes/CO2ken.json");
+const OmenDisputeResolutionInfo = require("./schemes/OmenDisputeResolution.json");
```
Comment (orenyodfat) [inline src/genericSchemeRegistry/index.ts:15]: OmenDisputeResolutionInfo  -> omenDisputeResolutionInfo

**Comment by orenyodfat** [comment]
is there any test dao with such generic scheme ? rinkeby  ?

**Comment by nicoelzer** [comment]
> is there any test dao with such generic scheme ? rinkeby ?

@orenyodfat it's added on this DAO: https://alchemy-omengenericsche-m5ml19.herokuapp.com/dao/0xd94c5810a43ae1560569592e041bf839e490cd1b/

File: src/genericSchemeRegistry/index.ts (hunk 1 above)
Comment (orenyodfat) [reply src/genericSchemeRegistry/index.ts:15]: @nicoelzer ?

Comment (nicoelzer) [reply src/genericSchemeRegistry/index.ts]: Hey @orenyodfat haven't seen your last comment, just pushed an update that fixes this.

**Pull request closed without merging**